decisions/wc_society_decisions.txt

```diff
@@ -542,5 +542,4 @@ society_decisions = {
 		# ai_will_do = {
 			# factor = 0
 		# }
-	# }
-}
```

Comment:
Nani? Why it's deleted?

Reply:
meme did, but if i see it correctly, deleted lines got added back in the same commit

Reply:
b2ea1b1a03f552e5b9851ebcd9870eefe61d20c1